core/parentchain/light-client/src/error.rs

```diff
@@ -37,6 +37,8 @@ pub enum JustificationError {
 
 #[derive(Debug, thiserror::Error)]
 pub enum Error {
+	#[error("Genesis header not found")]
+	NoGenesis,
```

Comment:
Not sure how much an effort would be to write a unit test for this new error, do you think it would be beneficial?

Reply:
Haha, this should not be here yet. I only start using it in the next PR. :) I will just remove it.